command/server.go

```diff
@@ -949,7 +960,17 @@ func (c *ServerCommand) Run(args []string) int {
 
 		namedSDLogger := c.logger.Named("service_registration." + config.ServiceRegistration.Type)
 		allLoggers = append(allLoggers, namedSDLogger)
-		configSR, err = sdFactory(config.ServiceRegistration.Config, namedSDLogger)
+
+		// Since we haven't even begun starting Vault's core yet,
+		// we know that Vault is in its pre-running state.
+		state := &sr.State{
+			VaultVersion:         version.GetVersion().VersionNumber(),
+			IsInitialized:        false,
+			IsSealed:             true,
+			IsActive:             false,
+			IsPerformanceStandby: true,
```

Comment:
This is probably correct, but it looks sort of odd to just assume that we are a performance standby.  Maybe there is a way to make this a little clearer, but I'm not sure how.

Reply:
I looked at the healthcheck and found that this is false when starting up, so have updated this to match.